### PR TITLE
Fetch & parse sourceMaps to check for @babel transpiler

### PR DESCRIPTION
**Pull request by kevinfarrugia**
Checking for `@babel` by parsing and downloading the source maps. Limited to pages which have publicly available source maps.

Test 1 (with public sourcemaps): https://webpagetest.httparchive.org/result/220529_QN_1J/1/details/#waterfall_view_step1

```
  "sourceMaps": {
    "count": 4,
    "ext": {
      "scss": 123,
      "js": 541,
      "svg": 13,
      "jpg": 1,
      "png": 7,
      "jsx": 152
    },
    "babel": true,
    "js": { "size": 1765807, "stats": {} }
  }
```

Test 2 (without public sourcemaps): https://webpagetest.httparchive.org/result/220529_37_1K/

```
  "sourceMaps": {
    "count": 17,
    "ext": {},
    "js": { "size": 0, "stats": {} }
  }
```

Test 3 (without sourcemaps): https://webpagetest.httparchive.org/result/220529_FV_1M/

```
{ "sourceMaps": { "count": 0, "ext": {} } }
```

### CODE REVIEW

File: dist/javascript.js [hunk 1]
```diff
@@ -3,94 +3,266 @@
...
+          }
+        }
+
+        if (url.pathname.match(/@babel/)) {
+            babel++;
+        }
+
+        return url;
+      });
+
```
Comment (rviscomi) [inline dist/javascript.js:97]: fix linter
```suggestion

```

File: dist/javascript.js [hunk 2]
```diff
@@ -3,94 +3,266 @@
...
+    setTimeout(() => {controller.abort()}, 5000);
+    return fetch(url, {signal: controller.signal});
+  }
+
 const requests = $WPT_REQUESTS;
 
-return JSON.stringify({
+let getSourceMaps = (async () => {
+  const $$ = (s) => [...document.querySelectorAll(s)];
+  const sourcemapRegex = /\/\/[#@] sourceMappingURL=(.+?)$/;
```
Comment (rviscomi) [inline dist/javascript.js:16]: Do you also want to capture the [`SourceMap`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/SourceMap) header?

Comment (kevinfarrugia) [reply dist/javascript.js:16]: Is there a custom-metric already looking at the SourceMap header? From last year's chapter, usage is low with 0.1% of pages using this.

https://almanac.httparchive.org/en/2021/javascript#source-maps

Comment (rviscomi) [reply dist/javascript.js:16]: Here's an example:

https://github.com/HTTPArchive/custom-metrics/blob/main/dist/responsive_images.js#L458

I agree usage is probably too low to matter, so I'll leave it up to you.

**Review by rviscomi — APPROVED** [review]
Will merge as-is to ensure these changes make it into the crawl. Open a new PR if needed.

File: dist/javascript.js (hunk 2 above)
Comment (kevinfarrugia) [reply dist/javascript.js:16]: Yeah, will leave out for now.

**Pull request merged**